Objective-C/CBLDocument.mm

```diff
@@ -210,7 +210,7 @@ - (void) replaceC4Doc: (CBLC4Document*)c4doc {
 
 #pragma mark - Fleece Encoding
 
-- (FLSliceResult) encode: (NSError**)outError {
+- (FLSliceResult) encode: (C4RevisionFlags*)outRevFlags error:(NSError**)outError {
```

Comment:
Should we name the method like `(FLSliceResult) encodeWithRevFlags: (C4RevisionFlags*)outRevFlags error:(NSError**)outError`?

Reply:
Also can you add some description about what the encode method is doing with the outRevFlags?